helm/alfresco-content-services/charts/activemq/values.yaml

```diff
@@ -57,9 +57,14 @@ livenessProbe:
   timeoutSeconds: 10
   failureThreshold: 1
 persistence:
-  existingClaim: "alfresco-volume-claim"
-  mountPath: "/opt/activemq/data"
-  subPath: "alfresco-infrastructure/activemq-data"
-  #chown doesn't work with dynamic provisioning so far
-  #issue on github: https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/300
-  chownWithDynamicProvisioning: false
+  enabled: true
+  VolumeSizeRequest: 10Gi
+  accessModes:
+    - ReadWriteOnce
+  storageClass:
+  data:
+    mountPath: "/opt/activemq/data"
+    subPath: "alfresco-infrastructure/activemq-data"
+      #chown doesn't work with dynamic provisioning so far
+      #issue on github: https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/300
+      chownWithDynamicProvisioning: false
```

Comment:
we should get rid of that and instead use a `securityContext` in the deployment.
As in https://github.com/Alfresco/acs-deployment/blob/641afabac300266ee573acc3b7864c3e0b824f43/helm/alfresco-content-services/templates/deployment-filestore.yaml#L51
(with different uid=33031 & gid=1000 see https://github.com/Alfresco/alfresco-docker-activemq/blob/master/Dockerfile)